package.json

```diff
@@ -19,19 +19,22 @@
     "react-component"
   ],
   "peerDependencies": {
-    "react": "~0.13.3"
+    "react": "~0.14.6"
   },
   "dependencies": {
     "classnames": "^1.2.0",
     "es5-shim": "^4.0.3",
+    "fbjs": "^0.6.1",
```

Comment:
Thanks for the commit guys. We'll take a look through that and hopefully get this merged to master asap